packages/runtime-common/card-document.ts

```diff
@@ -339,3 +340,18 @@ export function transformResultsToPrerenderedCardsDoc(results: {
     meta,
   };
 }
+
+export function transformResultsToCardTypeSummaryDoc(
+  results: CardTypeSummary[],
```

Comment:
from this context it's not clear what the "results" are the result of. let's just name this param `summary`